### PR TITLE
test: 💍 add test

### PR DESCRIPTION
**Pull request by SuperHuangXu**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [-] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [-] Tests for the changes have been added (for bug fixes / features)
- [-] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[-] CI related changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[-] No
```

## Other information

### CODE REVIEW

**Review by jmcdo29 — APPROVED** [review]

File: sample/19-auth-jwt/src/auth/auth.service.spec.ts [hunk 1]
```diff
@@ -29,4 +29,19 @@ describe('AuthService', () => {
...
+
+  it('should validateUser ok', async () => {
+    const res = await service.validateUser('maria','guess')
+    expect(res.userId).toEqual(3)
+  });
+
+  it('should validateUser fail', async () => {
+    const res = await service.validateUser('xxx','xxx')
+    expect(res).toBeNull()
+  });
```
Comment (kamilmysliwiec) [inline sample/19-auth-jwt/src/auth/auth.service.spec.ts:41]: Can we wrap it within a `describe("valideteUser")` block and then inside have to scenarios covered:

- should return a user object when credentials are valid
- should return null when credentials are invalid

And likewise below with the `login()` method

Comment (kamilmysliwiec) [reply sample/19-auth-jwt/src/auth/auth.service.spec.ts:41]: ping 🏓

**Comment by kamilmysliwiec** [comment]
Can you please rebase your branch? It shows 101 changed files now

**Comment by SuperHuangXu** [comment]
> Can you please rebase your branch? It shows 101 changed files now

Sorry, I am not very familiar with Git operations...

**Comment by kamilmysliwiec** [comment]
@SuperHuangXu maybe you can re-create your branch in this case? Cherry-pick your changes on a new branch and submit a new PR?

**Comment by SuperHuangXu** [comment]
> @SuperHuangXu maybe you can re-create your branch in this case? Cherry-pick your changes on a new branch and submit a new PR?

thanks~  this is [new PR](https://github.com/nestjs/nest/pull/6303)

**Pull request closed without merging**